python/cudf/cudf/tests/test_concat.py

```diff
@@ -1032,9 +1032,6 @@ def test_concat_join_no_overlapping_columns_many_and_empty(
                 ),
                 None,
             ],
-            marks=pytest.mark.xfail(
```

Comment:
I guess this has just been xpassing for a while?

Reply:
Yep :)